tests/jax_jit_test.py

```diff
@@ -199,10 +200,11 @@ def test_arg_signature_of_value(self):
     self.assertTrue(signature.weak_type)
 
   def test_signature_support(self):
+    jit = partial(api._jit, "cpp")
```

Comment:
This looks incorrect — `api._jit` now takes a bool as its first argument.

Reply:
You're absolutely right. Apologies for missing this!